crest/Assets/Crest/Crest/Scripts/Helpers/FloatingOrigin.cs

```diff
@@ -65,35 +69,149 @@ public class FloatingOrigin : MonoBehaviour
         [Tooltip("Optionally provide a list of Gerstner components to avoid doing a FindObjectsOfType() call."), SerializeField]
         ShapeGerstnerBatched[] _overrideGerstnerList = null;
 
+        [Space(10)]
```

Comment:
this is good. i should probably look at adding spaces to try make OceanRenderer ui a bit clearer